lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -90,9 +90,15 @@ our @ISA = qw/Text::Markdown/;
 sub _DoAutoLinks {
     my ($self, $text) = @_;
 
+    # auto-replace every http(s) reference which is not already either html
+    # 'a href...' or markdown link '[link](url)'
+    $text =~ s{(?<!['"(])(http[s]?://[^\s]*)}{<a href="$1">$1</a>}gi;
```

Comment:
yes, you are right, I didn't know this. https://github.com/bobtfish/text-markdown/blob/41a21bebb74e42e0bfaa6a248b3884077c184ba7/lib/Text/Markdown.pm#L1454 should do it although my observations show otherwise.

I tested:
 - Not including my replacement: openQA server does not render links clicklable
 - Including: it does

This also works for existing comments as they are still stored as the text exactly as I wrote so we can easily test by restarting just openQA and check the rendering of the existing comments.

I am trying to understand why the Text::Markdown code does not seem to work (for me/us). In the meantime, as commented in the PR, I would skip this commit but still keep the others if you'd prefer.

BTW:
Replacing a URL by a href plus the text is AFAIK no standard markdown.

In markdown links still would need to be marked, eg. like:
`[this](http://www.example.org)`
yielding
[this](http://www.example.org)
but http://www.example.org is only replaced by a clickable link because github flavored markdown does it, see [here](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) under "Automatic linking for URLs". It's not part of standard markdown.